easybuild/easyconfigs/g/GPAW/GPAW-0.11.0-customize.patch

```diff
@@ -0,0 +1,62 @@
+Patch the customize.py file to pick up EasyBuild env variables for
+various dependencies.
+
+Originally written by mikstr in PR 4400.
```

Comment:
@mikstr another variant of the patch to check, seems like an extended version of yours in #4400?